api/v1alpha4/types.go

```diff
@@ -116,6 +120,26 @@ type SubnetFilter struct {
 	NotTagsAny      string `json:"notTagsAny,omitempty"`
 }
 
+type PortOpts struct {
+	NetworkID           string              `json:"networkID" required:"true"`
```

Comment:
Since `NetworkID` is a mandatory field, would it make sense if `ports:` were children of the list items of `networks:`?